### PR TITLE
Add criteria parameter which indicates whether distance sorting is used.

### PR DESCRIPTION
**Pull request by Rosejoycrocker**
- adds new integer parameter 'use_dist' which =1 when distance sorting is to be used and 0 when distance sorting is not to be used
- addresses issue #238

### CODE REVIEW

File: src/ecosystem/Ecosystem.jl [hunk 1]
```diff
@@ -106,6 +106,8 @@ Base.@kwdef struct Criteria{P,N} <: EcoModel
         name="Low Area Tolerance", description="Tolerance for low proportional space for seeding deployments.")
     deployed_coral_risk_tol::P = Param(1.0, ptype="real", bounds=(0.75, 1.0), dists="unif",
         name="Risk Tolerance", description="Filters out sites with heat/wave stress above threshold.")
+    use_dist::N = Param(0, ptype="integer", bounds=(0, 1 + 1), dists="unif",
+        name="Use Distance", description="Turns using distance sorting on and off.")
```
Comment (ConnectedSystems) [inline src/ecosystem/Ecosystem.jl:110]: How about "Distance Threshold" as the name?

For the description, how about: "Turns ~using~ distance sorting on **or** off"

**Review by ConnectedSystems — CHANGES_REQUESTED** [review]
Thanks Rose, requesting a small change regarding the factor name/description

File: src/ecosystem/Ecosystem.jl (hunk 1 above)
Comment (Rosejoycrocker) [reply src/ecosystem/Ecosystem.jl:110]: Sure, sounds good. Are these used for presenting analysis in a more understandable format?

Comment (ConnectedSystems) [reply src/ecosystem/Ecosystem.jl:110]: Yes

Comment (Rosejoycrocker) [reply src/ecosystem/Ecosystem.jl]: Ok, cool

**Review by ConnectedSystems — APPROVED** [review]

**Pull request merged**